modules/govuk/manifests/apps/signon.pp

```diff
@@ -37,7 +37,7 @@
     port               => $port,
     vhost_ssl_only     => true,
     health_check_path  => '/users/sign_in',
-    log_format_is_json => true,
+    legacy_logging     => false,
     vhost_aliases      => ['signonotron'],
     logstream          => $ensure_logstream,
```

Comment:
The `logstream` param on `govuk::app` defaults to true so I think we can delete the param from this class and from hieradata entirely.